packages/subgraph/src/entities/StabilityDeposit.ts

```diff
@@ -20,7 +20,7 @@ function getStabilityDeposit(_user: Address): StabilityDeposit {
 
     newStabilityDeposit.owner = owner.id;
     newStabilityDeposit.depositedAmount = DECIMAL_ZERO;
-
+    newStabilityDeposit.frontend = owner.frontend;
```

Comment:
This only sets the frontend once, when the first deposit is made by the user. However, if they later withdraw everything, and make a new deposit through a different frontend, this code won't be invoked.